src/components/TodoListFooter.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./TodoListFooter.css";
+
+class TodoListFooter extends React.Component {
```

Comment:
Check if this component is rendering when you change a caption of an item. It should rerender only when you update the length of the todo list or change the state of an item to active/completed.